src/menu.tsx

```diff
@@ -91,38 +92,49 @@ export function get_menu_items(
             }
         );
     } else if (selected.length === 1) {
+        const item = selected[0];
+
+        if (item.type === 'reg' && item.size && item.size < 1000000) // 1MB
+            menuItems.push(
+                {
+                    id: "open-file",
```

Comment:
And dito here -- this is "edit". There's other ways how you can "open" a file (image viewer, run executable, render HTML in a new tab, etc.)

Reply:
Well there is a "View" and "Edit" mode, depending on the permissions Edit or View is shown using `test -w` ideally we check this upfront but I think we discussed this a million times and this is imo not easy with static information.

So do we cockpit.spawn `test -w` when opening a context menu to determine what to show or do we keep a generic text?